lib/model.js

```diff
@@ -443,7 +451,11 @@ module.exports = function(registry) {
       accepts: {arg: 'refresh', type: 'boolean', http: {source: 'query'}},
       accessType: 'READ',
       description: 'Fetches belongsTo relation ' + relationName + '.',
-      returns: {arg: relationName, type: modelName, root: true}
+      returns: {arg: relationName, type: modelName, root: true},
+      authorization: function(ctx, done) {
+        var targetSharedMethod = relation.modelTo.sharedClass.find('findOne', true);
+        relation.modelTo.checkAccess(ctx.accessToken, ctx.modelId, targetSharedMethod, ctx, done);
```

Comment:
I think you need to call `accessRequest.isAllowed()` too, otherwise the statement from PR description is not accurate, in particular "checkAccess() for customer with id 1" is not called. Unless I am mistaken?